docs/Customization/Dsptools-Blocks.rst

```diff
@@ -0,0 +1,105 @@
+.. _dsptools-blocks:
+
+Dsptools Blocks
```

Comment:
There needs to be some introduction here. 
What is the main use-case here? (DSP chains....)
Why does this main use-case need a different flow than the standard MMIO method?